prod/php/ElasticOTel/HttpTransport/ElasticHttpTransport.php

```diff
@@ -56,19 +59,32 @@ public function __construct(
         $this->endpoint = $endpoint;
         $this->contentType = $contentType;
 
-        initialize($endpoint, $contentType, $headers, $timeout, $retryDelay, $maxRetries);
+        /**
+         * \Elastic\OTel\HttpTransport\* functions are provided by the extension
+         *
+         * @noinspection PhpUnnecessaryFullyQualifiedNameInspection, PhpUndefinedFunctionInspection
+         */
+        \Elastic\OTel\HttpTransport\initialize($endpoint, $contentType, $headers, $timeout, $retryDelay, $maxRetries); // @phpstan-ignore function.notFound
     }
 
     public function contentType(): string
     {
         return $this->contentType;
     }
 
+    /**
+     * @return FutureInterface<string>
+     */
     public function send(string $payload, ?CancellationInterface $cancellation = null): FutureInterface
     {
-        enqueue($this->endpoint, $payload);
+        /**
+         * \Elastic\OTel\HttpTransport\* functions are provided by the extension
+         *
+         * @noinspection PhpUnnecessaryFullyQualifiedNameInspection, PhpUndefinedFunctionInspection
+         */
+        \Elastic\OTel\HttpTransport\enqueue($this->endpoint, $payload); // @phpstan-ignore function.notFound
 
-        return new CompletedFuture(null);
+        return new CompletedFuture($payload);
```

Comment:
why payload is passed?
in otel exporters response body is passed there, we can't do this because it is queued to be sent in another thread